pkg/analyser/analysis.go

```diff
@@ -64,6 +64,7 @@ type serializableAnalysis struct {
 	Alerts          map[string][]alerter.SerializableAlert `json:"alerts"`
 	ProviderName    string                                 `json:"provider_name"`
 	ProviderVersion string                                 `json:"provider_version"`
+	Duration        int                                    `json:"scan_duration"`
```

Comment:
Can you add `omitempty` and name the field `ScanDuration` for consistency with the json field name ?

It's also an `uint`